tests/test_codex_feed_hooks.py

```
@@ -420,6 +420,138 @@ def test_codex_monitor_exits_when_workspace_has_no_surfaces(cli_path: str, root:
+            ],
+            capture_output=True,
+            text=True,
+            check=False,
+            env=env,
+            timeout=3,
+        )
+    if result.returncode != 0:
+        raise AssertionError(
+            f"hooks codex monitor failed exit={result.returncode}\n"
+            f"stdout={result.stdout}\nstderr={result.stderr}"
+        )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)